Added recent changes and moved byIdAndDate to new profile (#565)
admin-shell-io/aas-specs-api: documentation/IDTA-01002-3/modules/ROOT/pages/specification/interfaces.adoc

Reviewer: ```suggestion
e|xref:GetAssetAdministrationShellVersionByIdAndDate[GetAssetAdministrationShellVersionByIdAndDate] |Requested Asset Administration Shell at a specific point in time 4+a|Returns a specific Asset Administration Shell at a specific point in time.
```

2+|Interface: [[interface-AssetAdministrationShellRepository]]_Asset Administration Shell Repository_
h|Operation Name h|Description
e|xref:GetAllAssetAdministrationShells[GetAllAssetAdministrationShells] |Returns all Asset Administration Shells
changed Asset Administration Shells

> e|xref:GetAllAssetAdministrationShellRecentChanges[GetAllAssetAdministrationShellRecentChanges] |Returns information about recently

Reviewer 2: ```suggestion
e|xref:GetAllAssetAdministrationShells[GetAllAssetAdministrationShells] |Returns all Asset Administration Shells
e|xref:GetAllAssetAdministrationShellsRecentChanges[GetAllAssetAdministrationShellsRecentChanges] |Returns information about recently changed Asset Administration Shells
```

e|payload a|List of Asset Administration Shells |no |link:https://industrialdigitaltwin.io/aas-specifications/IDTA-01001/v3.1/spec-metamodel/core.html#AssetAdministrationShell[AssetAdministrationShell] |1..*
|===

==== Operation GetAssetAdministrationShellRecentChanges

[.table-with-appendix-table]
[width=100%,cols="25%,35%,10%,20%,10%"]
|===
h|Operation Name 4+e|[[GetAssetAdministrationShellRecentChanges]]GetAssetAdministrationShellRecentChanges
h|Explanation 4+a|Returns information about recently changed Asset Administration Shells
h|semanticId 4+|`\https://admin-shell.io/aas/API/GetAssetAdministrationShellRecentChanges/3/2`

h|Name h|Description h|Mand. h|Type h|Card.

5+h|Input Parameter

> e|key a|The key of the AssetId

Reviewer: Then why not call it assetId or globalAssetId? Key is not very clear.
In the opeanAPI you call it globalAssetId it seems to me

Author: Fixed

The name of the specific asset identifier or the predefined name “_globalAssetId_” that would refer to the _AssetInformation/globalAssetId_ |yes |string |1
e|createdAt a|Return Shells created at or later |no | xref:components/parameters/CreatedAt|1
e|updatedAt a|Return Shells updated at or later |no | xref:components/parameters/UpdatedAt|1
e|limit a|The maximum size of the result set |no |nonNegativeInteger |1
e|cursor a|The position from which to resume a result listing |no |string |1
5+h|Output Parameter
e|statusCode a|Status code |yes |xref:specification/interfaces-payload.adoc#StatusCode[StatusCode] |1
e|payload a|Requested Asset Administration Shell Change Infos |yes |xref:specification/interfaces-payload.adoc#AssetAdministrationShellRecentChange[AssetAdministrationShellRecentChange] |1
|===

> 

Reviewer: How do you get the complete AAS under consideration? All of them have the same ID, only different updateDates. 
Probably we need to add a constraint that createdAt is unique for an AAS with a specific ID... Same for Submodel etc.

Author: Birgit confirmed, that she didn't have the two uses cases in mind. I think that topic is solved.

e|payload a|List of Asset Administration Shells |no |link:https://industrialdigitaltwin.io/aas-specifications/IDTA-01001/v3.1/spec-metamodel/core.html#AssetAdministrationShell[AssetAdministrationShell] |1..*
|===

==== Operation GetAllAssetAdministrationShellRecentChanges

[.table-with-appendix-table]
[width=100%,cols="25%,35%,10%,20%,10%"]
|===
h|Operation Name 4+e|[[GetAllAssetAdministrationShellRecentChanges]]GetAllAssetAdministrationShellRecentChanges
h|Explanation 4+a|Returns information about recently changed Asset Administration Shells

> h|semanticId 4+|`\https://admin-shell.io/aas/API/GetAllAssetAdministrationShellRecentChanges/3/2`

Reviewer 2: ```suggestion
==== Operation GetAllAssetAdministrationShellsRecentChanges

[.table-with-appendix-table]
[width=100%,cols="25%,35%,10%,20%,10%"]
|===
h|Operation Name 4+e|[[GetAllAssetAdministrationShellsRecentChanges]]GetAllAssetAdministrationShellsRecentChanges
h|Explanation 4+a|Returns information about recently changed Asset Administration Shells
h|semanticId 4+|`\https://admin-shell.io/aas/API/GetAllAssetAdministrationShellsRecentChanges/3/2`
```
